apps/web-giddh/src/app/theme/ng-social-login-module/providers/google-login-provider.ts

```diff
@@ -58,7 +58,7 @@ export class GoogleLoginProvider extends BaseLoginProvider {
         });
     }
 
-    public signOut(): Promise<any> {
+    public signOut(): Promise<void> {
```

Comment:
if function has return statement, why are we changing any to void?

Reply:
`void` is provided because `resolve();` is not passing any data when Promise is getting resolved